ShenkinStore/Controllers/ShoppingCartController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ShenkinStore.Models;
+using ShenkinStore.ViewModels;
+
+namespace ShenkinStore.Controllers
+{
+    public class ShoppingCartController : Controller
+    {
+        ShenkinContext db = new ShenkinContext();
+
+        public string shoppingCartId { get; set; }
+
+
+        // GET: ShoppingCart
+
+        public ActionResult Index()
+        {
+         
+            var userID = HttpContext.Session.GetString("UserID");
+            
+            
```

Comment:
Remove the unnecessary line breaks, keep 1 line break or none between lines.
The close rows should be code that's related to one another and the break line should separate between different actions.

Reply:
done